torch/_C/_distributed_rpc.pyi

```diff
@@ -79,7 +78,8 @@ class _TensorPipeRpcBackendOptionsBase(RpcBackendOptions):
         rpc_timeout: float = _DEFAULT_RPC_TIMEOUT_SEC,
         init_method: str = _DEFAULT_INIT_METHOD,
         device_maps: Dict[str, Dict[torch.device, torch.device]] = {},
-        devices: List[torch.device] = list()): ...
```

Comment:
We should not use mutable variables as function defaults.

Ref: [dangerous-default-value / W0102](https://pylint.pycqa.org/en/latest/user_guide/messages/warning/dangerous-default-value.html)

Correct code:

```python
def func(..., devices: Optional[List[torch.device]] = None):
    if devices is None:
        devices = []
    ...
```